pysiaf/__init__.py

```diff
@@ -20,8 +14,4 @@
 from .tests import test_aperture#, test_polynomial
```

Comment:
So this line is the one of concern:
```
from .tests import test_aperture
```
That means even to just `import pysiaf`, it has to also `import pysiaf.tests`, which in turn imports `pytest`. This means that users need to have `pytest` installed in order to import `pysiaf`. However `pytest` is not listed as a requirement in setup.py, etc. 

Two suggested paths:
1. Remove that import line, if you don't need to have `test_aperture` imported at runtime. This is the easiest fix. 
2. Alternatively, add `pysiaf` to the requirements file.